deps/key-value-storage/src/lib.rs

```diff
@@ -12,11 +12,15 @@ use std::sync::Arc;
 pub mod error;
 pub use error::{KeyValueStorageError, Result};
 
-pub mod simple;
+pub mod memory;
 
 #[cfg(feature = "postgres")]
 pub mod postgres;
 
+pub mod local_json;
+
+pub mod local_fs;
+
 #[derive(Default)]
 pub struct SetParameters {
```

Comment:
I don't really like this struct, especially since it only has one field. We could just pass an overwrite argument to `set`. The logic around overwriting stuff is a little convoluted anyway.

We can tweak this in a follow-up though, since it's not actually added in this PR.

Reply:
Oh yes. Originally there would be some other parameters than overwrite for the set context due to https://github.com/confidential-containers/trustee/pull/1000#discussion_r2444051093 and this is a room for them. But we can change it if we do not need this